pynvjitlink/patch.py

```diff
@@ -44,6 +48,24 @@
     Linker = object
 
 
+spec = importlib.util.find_spec("numba_cuda")
+if spec is not None:
+    _numba_cuda_in_use = True
+    _numba_cuda_error = "`numba_cuda` includes patches from pynvjitlink, so no further patches are needed. "
+
+    import numba_cuda
+
+    numba_cuda_ver = (int(x) for x in numba_cuda.__version__.split("."))
+    if numba_cuda_ver < (0, 2, 0):
+        suggestion = "Instead, use NUMBA_CUDA_ENABLE_PYNVJITLINK environment variable to enable pynvjitlink features."
```

Comment:
Can't you use `config.CUDA_ENABLE_PYNVJITLINK` for later versions as well?

Reply:
Yes, but I think the support is incomplete - the user would have to do the config prior to the first import line of `numba.cuda`.

Reply:
I think that was the case, but I think now as long as it's set before a kernel is jitted then it's OK - the config variable doesn't appear to be read until the linker is instantiated in `Linker.new()`.

Reply:
Anyway, I think the recommendation is OK as-is, no need to change anything.

Reply:
> I think that was the case, but I think now as long as it's set before a kernel is jitted then it's OK - the config variable doesn't appear to be read until the linker is instantiated in `Linker.new()`.

Ah, you are right about this. Sorry replied late on this.